deploy/helm/sumologic/conf/events/events.conf

```diff
@@ -30,12 +30,13 @@
   verify_ssl "#{ENV['VERIFY_SSL']}"
   proxy_uri "#{ENV['PROXY_URI']}"
   <buffer>
+    {{- if eq .Values.sumologic.fluentd.buffer "file" }}
+    @type file
+    path /fluentd/buffer/events
```

Comment:
@vsinghal13 is this path local to node? If yes, all pods inside that node will store buffer in the same file?

Reply:
<img width="809" alt="Screenshot 2019-12-18 at 8 48 25 PM" src="https://user-images.githubusercontent.com/8642245/71098480-c5e49a80-21d7-11ea-9015-13bad0eddda9.png">

Reply:
Yes, the path is local to node. Each buffer section needs to have a unique path. The pods will store will store in different paths based on the config path provided in the respective buffer section.